rapids-cmake/cmake/support_conda_env.cmake

```
@@ -164,6 +164,17 @@ function(rapids_cmake_support_conda_env target)
                             "$<HOST_LINK:SHELL:LINKER:-rpath-link=$ENV{BUILD_PREFIX}/lib>")
       endif()
 
+      # Make all paths like `$PREFIX/<path>` relative to `$PREFIX` so `<path>`.
+      target_compile_options(${target}
+                             INTERFACE "$<$<COMPILE_LANGUAGE:C>:-ffile-prefix-map=$ENV{PREFIX}/=''>"
+      )
+      target_compile_options(${target}
+                             INTERFACE "$<$<COMPILE_LANGUAGE:CXX>:-ffile-prefix-map=$ENV{PREFIX}/=''>"
+      )
+      target_compile_options(${target}
+                             INTERFACE "$<$<COMPILE_LANGUAGE:CUDA>:-Xcompiler=-ffile-prefix-map=$ENV{PREFIX}/=''>"
```

Minor nit, this should work as a single command.
```suggestion
                             INTERFACE "$<$<COMPILE_LANGUAGE:C>:-ffile-prefix-map=$ENV{PREFIX}/=''>"
                             "$<$<COMPILE_LANGUAGE:CXX>:-ffile-prefix-map=$ENV{PREFIX}/=''>"
                             "$<$<COMPILE_LANGUAGE:CUDA>:-Xcompiler=-ffile-prefix-map=$ENV{PREFIX}/=''>"
```

---

Sounds reasonable. Thanks Vyas! 🙏